R/HS_PGA_char.R

```diff
@@ -0,0 +1,25 @@
+#' Converts HS-PGA numerical scores to character values.
+#'
+#' @param hs_pga_scores Integer vector representing the HS-PGA scores (1: Clean, 2: Minimal, 3: Mild, 4: Moderate, 5: Severe, 6: Very Severe).
+#' @return A character vector representing the corresponding HS-PGA categories.
+#' @export
+#' @examples
+#' hs_pga_scores <- c(1, 2, 3, 4, 5, 6)
+#'
+#' hs_pga_char(hs_pga_scores)
+#'
+#' @import
```

Comment:
I believe that this is causing the `NAMESPACE` file issue.  You do not need to use `@import` unless you are using a function from another package all the time.  (I tend to avoid importing from other packages, in general.  And, I tend to use `packagename::functionname()` to make it clear when another package is being used.)